metrics/ethereum.py

```
@@ -141,6 +143,24 @@ def __init__(
         )
         self.labels.update_label(MetricLabelKey.API_METHOD, "eth_subscribe")
 
+    async def send_with_timeout(self, websocket, message: str, timeout: float) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `websocket`

(ANN001)